packages/flutter/lib/src/rendering/object.dart

```diff
@@ -74,6 +74,10 @@ class PaintingContext {
     assert(child.needsPaint);
     child._layer ??= new ContainerLayer();
     child._layer.removeAllChildren();
+    assert(() {
+      child._layer.debugOwner = child.debugOwner ?? child.runtimeType;
```

Comment:
I would have gone with `"${child.runtimeType}  ${child.debugOwner}"` or some such. But this is fine too.